cmd/log-collector/cmd/cmd_test.go

```diff
@@ -116,5 +118,15 @@ var _ = Describe("log collector cmd helper unit tests", func() {
 			err := logCollector.InitializeKubeClients()
 			Expect(err).ShouldNot(BeNil())
 		})
+
+		It("Should prioritize log collector KUBECONFIG flag value", func() {
+			logCollector.KubeConfig = internal.KubeConfigDefault
+			err := os.Setenv(internal.KubeconfigEnv, "")
```

Comment:
set this to some non-empty value to env variable.
Assign a different value to `logCollector.KubeConfig`.